tests/data/jsons/url_protocols.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "https://schemas.conda.io/menuinst-1.schema.json",
+    "menu_name": "Example with file type association",
+    "menu_items": [
+        {
+            "name": "FileTypeAssociation",
+            "icon": null,
+            "command": [
+                "{{ PYTHON }}",
+                "-c",
+                "import sys, pathlib as p; Path(r'__OUTPUT_FILE__').write_text(sys.argv[1])"
+            ],
+            "platforms": {
+                "win": {
+                    "command": [
+                        "{{ PYTHON }}",
+                        "-c",
+                        "import sys, pathlib as p; Path(r'__OUTPUT_FILE__').write_text('%1')"
+                    ]
+                },
+                "linux": {
+                    "command": [
+                        "{{ PYTHON }}",
+                        "-c",
+                        "import sys, pathlib as p; Path(r'__OUTPUT_FILE__').write_text('%u')"
+                    ],
+                    "MimeType": ["x-scheme-handler/menuinst"]
+                },
+                "osx": {
+                    "command": [
+                        "{{ PYTHON }}",
+                        "-c",
+                        "import sys, pathlib as p; Path(r'__OUTPUT_FILE__').write_text(sys.argv[1])"
+                    ],
+                    "CFBundleURLTypes": [
+                        {
+                            "CFBundleURLIconFile": "{{ MENU_DIR }}/my_protocol_handler",
+                            "CFBundleURLName": "my-protocol-handler.menuinst",
+                            "CFBundleTypeRole": "Viewer",
```

Comment:
For this scheme to work properly I think the `CFBundleTypeRole` actually needs to be omitted. This took me hours to figure out. I was so desperate to open a thread on Apple's developer forum: https://developer.apple.com/forums/thread/725410

Reply:
Oh, nice, thanks. I added this to the docs.